test/cli_volume_test.go

```diff
@@ -327,4 +327,44 @@ func (suite *PouchVolumeSuite) TestVolumeListOptions(c *check.C) {
 			}
 		}
 	}
+
+	// test filter options
+	volumeName4 := "volume_" + funcname + "4"
+	command.PouchRun("volume", "create", "--name", volumeName4, "--driver", "local", "--label", "test=foo").Assert(c, icmd.Success)
+	defer command.PouchRun("volume", "rm", volumeName4)
+	// test name filter
+	res := command.PouchRun("volume", "list", "--filter", "name="+volumeName4)
+	res.Assert(c, icmd.Success)
+
+	found := 0
```

Comment:
could we reorder the cases here? for example, use test case table to do that. I think there are a lot of duplicate code here. thanks.

Reply:
thanks for remind me, update.